docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+  particularly the kernel related packages
+
+- Feel a sense of personal responsibility for the quality of Ubuntu releases
+  and for the satisfaction of Ubuntu users
+
+- Exercise great care in their work, with the understanding that their efforts
+  have a direct impact on others, including:
+
+  - every Ubuntu user
+  - the Ubuntu release team
+  - corporate partners who provide support for Ubuntu
+
+Application Process
+===================
+
+As alluded to in the member profile above, membership consideration for the
+ubuntu-kernel-uploaders team adheres to a strict policy. Anyone considering
+applying should meet **all** of the criteria listed below:
+
+1. Have a history of substantial and direct contribution to the distribution;
+   particularly the kernel related packages
+
+2. A thorough understanding of the Ubuntu kernel patch submission process
+
+   a. Demonstrates an understanding of this process by having submitted
+      multiple patches which were accepted over a six-month development cycle
+   b. Demonstrates an understanding of this process by having reviewed and
+      Ack'd/Nack'd multiple patches over a six-month development cycle 
+
+3. A thorough understanding of the Ubuntu release cycle and associated
+   milestone and freeze dates
+
+4. A thorough understanding of the Ubuntu Kernel SRU cadence
+
+5. A thorough understanding of the upstream kernel development cycle and how it
+   relates to the Ubuntu kernel development cycle
+
+6. Demonstrate a chain of trust by having multiple sponsored kernel uploads
+   over a six-month development cycle by various existing members of the
+   ubuntu-kernel-uploaders team.
+
+If you are not an official ubuntu-kernel-uploaders member yet, but fulfill all
+of the criteria above, you are likely a promising candidate for joining the
+team. If you are interested in joining, please start by preparing your
+application. Please use the following template for your application:
+
+https://wiki.ubuntu.com/Kernel/Dev/PPUApplicationTemplate
+
+An example application can also be seen at the following: 
+
+https://wiki.ubuntu.com/LuisHenriques/PerPackageUploaderApplication 
+
+At least three existing ubuntu-kernel-uploaders members have to confirm that
+they have worked enough with you to judge your skills and that you meet the
+criteria above. Usually these three people are your sponsors.
+
+Screening Process
+=================
+
+Once your application has been prepared and you are ready to be screened,
+please send an email to the `Ubuntu kernel-team mailing list`_ requesting your
+application be reviewed.
+
+You will then get a notification from the team about the scheduled IRC meeting
+(#ubuntu-kernel channel at irc.libera.chat) where you will be interviewed and a
+vote regarding your membership will be taken. 
+
+As part of the interview you will be asked to briefly introduce yourself, so
+please prepare a 2-3 line introduction beforehand to speed up the process. Only
```

Comment:
```suggestion
prepare a 2-3 line introduction beforehand to speed up the process. Only
```

Same as previous comment; since this is a reference guide we can drop the 'please'.